decidim-blogs/app/forms/decidim/blogs/admin/post_form.rb

```diff
@@ -10,24 +10,45 @@ class PostForm < Decidim::Form
         translatable_attribute :title, String
         translatable_attribute :body, String
 
-        attribute :user_group_id, Integer
+        attribute :decidim_author_id, Integer
+        attribute :id, Integer
```

Comment:
I believe the `id` attribute is unnecessary here because of:
https://github.com/decidim/decidim/blob/95dc6673425d5b7f2d30f3b0c24dcda8aeb5d88a/decidim-core/lib/decidim/attribute_object/model.rb#L30

Unless there's a particular reason for this to be forced as an integer (I don't think there is)?

It's not forced as integer by default because I remember there were some cases where we have string IDs.

Reply:
Makes sense, I don't think there isn't any reason for doing this here. I removed it.